actors/datacap/tests/harness/mod.rs

```diff
@@ -52,6 +54,66 @@ impl Harness {
         assert_eq!(self.registry, state.registry);
     }
 
+    pub fn mint(
+        &self,
+        rt: &mut MockRuntime,
+        to: &Address,
+        amount: &TokenAmount,
+        operators: Vec<Address>,
+    ) -> Result<(), ActorError> {
+        rt.expect_validate_caller_addr(vec![*VERIFIED_REGISTRY_ACTOR_ADDR]);
+
+        // Expect the token receiver hook to be called.
+        let hook_params = TokensReceivedParams {
+            from: DATACAP_TOKEN_ACTOR_ADDR.id().unwrap(),
+            to: to.id().unwrap(),
+            operator: VERIFIED_REGISTRY_ACTOR_ADDR.id().unwrap(),
+            amount: amount.clone(),
+            operator_data: Default::default(),
+            token_data: Default::default(),
+        };
+        rt.expect_send(
```

Comment:
I might be confused about this but I don't think this ExitCode::OK is a good representation of what will actually happen.  The receiver hook gets called on the verified client which for the time being will be an actor that can sign things which means it will not implement the receiver hook method and will fail with INVALID_METHOD or something like that.

I think that everything still works because of your impl of Token Messaging trait and I want to see that exercised.

Reply:
> the verified client which for the time being will be an actor that can sign things which means it will not implement the receiver hook method

Your confusion is that the built-in account and multisig actors do implement the receiver hook (#555), which is independent of any signing/validation-related things.

Reply:
aha cool !